aggregators/1.0/modules/ROOT/pages/aggregators-sync-async.adoc

```diff
@@ -0,0 +1,35 @@
+= Synchronous Versus Asynchronous Aggregations
+
+An aggregation is complete based on a new element added to the list, as when a max size is specified, or because some timeout or time period is complete. This is important because the type of aggregation determines which chain of components to execute with the list of elements.
+There are two kinds of triggers for an aggregation completion: synchronous and asynchronous.
+
+[IMPORTANT]
+Each time counter associated with an aggregator starts counting from the moment the first message of a group arrives. Once the aggregation is complete, the counter resets and waits for the next element to arrive. +
+For single-group aggregators (time-based and size-based aggregators scopes), there is only one time counter, but for the group-based aggregator, there is one counter per group.
+
+== Synchronous Aggregations
+
+For aggregations completed because a new element arrives (Sync), at least one of the following things occur:
+
+* If the aggregator scope is configured with an *Aggregation complete* route, the components inside that route execute with the payload being the list of aggregated elements. +
+* If the aggregator scope has a referenced *Aggregator listener*, the flow to which that listener belongs executes with the payload being the list of aggregated elements.
+
+== Asynchronous Aggregations
+
+For aggregations completed due to a time period or timeout being reached (Async), the only thing that can happen is that if the aggregator scope has a referenced *Aggregator listener*, and the listener accepts timed-out aggregations, the flow to which that listener belongs executes with the payload being the list of aggregated elements. +
```

Comment:
Slight rewording - I was trying to make this shorter, but haven't succeeded. 

For aggregations completed because a time period ended or a timeout was reached (Async), the only thing that can happen is that if the aggregator scope has a referenced *Aggregator listener* and the listener accepts timed-out aggregations, the flow to which that listener belongs executes the list of aggregated elements as the payload.